packages/wallet/backend/src/user/service.ts

```diff
@@ -109,6 +118,91 @@ export class UserService implements IUserService {
 
     return !!user
   }
+  public async createDefaultAccount() {
+    const existingUser = await this.getByEmail(
+      this.env.DEFAULT_WALLET_ACCOUNT.email
+    )
+
+    if (existingUser) return
+
+    const asset = await this.rafikiClient.getRafikiAsset('USD', 2)
+    if (!asset) await this.rafikiClient.createAsset('USD', 2)
+    const defaultWalletUser = this.env.DEFAULT_WALLET_ACCOUNT
+    const defaultBoutiqueUser = this.env.DEFAULT_BOUTIQUE_ACCOUNT
+
+    const { createdWalletUser, defaultAccount } =
+      await this.createWalletDefaultUser(defaultWalletUser)
+    const { createdBoutiqueUser, defaultBoutiqueAccount } =
+      await this.createBoutiqueDefaultUser(defaultBoutiqueUser)
+
+    if (defaultAccount && defaultBoutiqueAccount) {
+      const typedArray = new Uint32Array(1)
+      getRandomValues(typedArray)
+
+      await this.walletAddressService.create({
+        accountId: defaultAccount.id,
+        walletAddressName: typedArray[0].toString(16),
+        publicName: 'Default Payment Pointer',
+        userId: createdWalletUser.id,
+        isWM: false
+      })
+
+      const boutiqueWallet = await this.walletAddressService.create({
+        accountId: defaultBoutiqueAccount.id,
+        walletAddressName: 'boutique',
+        publicName: 'Default Payment Pointer',
+        userId: createdBoutiqueUser.id,
+        isWM: false
+      })
+
+      await this.walletAddressService.registerKey(
+        createdBoutiqueUser.id,
+        defaultBoutiqueAccount.id,
+        boutiqueWallet.id,
+        {
+          publicKeyPEM: this.env.DEFAULT_BOUTIQUE_KEYS.public_key,
+          privateKeyPEM: this.env.DEFAULT_BOUTIQUE_KEYS.private_key,
+          keyId: this.env.DEFAULT_BOUTIQUE_KEYS.key_id
+        }
+      )
+    }
+
+    this.logger.info('Default users have been successfully created')
+  }
+
+  private async createBoutiqueDefaultUser(
```

Comment:
both methods do the same thing.
as below only the user id is used form user info and `userId` exists in account details, returning the new account should be enough